packages/mdc-textfield/README.md

```diff
@@ -373,7 +373,12 @@ being used as the text field's "helper text". It defaults to `null`. If the text
 contains an `aria-controls` attribute on instantiation of the component, it will look for an element
 with the corresponding id within the document and automatically assign it to this property.
 
-##### MDCTextField.disabled
+##### MDCTextfield.helperTextContent
```

Comment:
Another renaming thing due to PR#1485 (sorry!)....MDCTextfield should be MDCTextField. Same for below.

Reply:
Good eye! Fixed with af16d8d